### DISCUSSION TITLE
Need help on loading a customized bert sentence embedding model in onnx format

### DISCUSSION THREAD

**Discussion by eedreamer** [JohnSnowLabs/spark-nlp#14434] [Q&A]
Anyone knows if spark NLP supports to load a customized bert sentence embedding model trained in pytorch and converted into onnx format?

**Comment by maziyarpanahi** [comment]
As long as your model is compatible with the `transformers` library or it can be loaded in `optimum` library you can import it easily. You have all the notebooks here to guide you: https://github.com/JohnSnowLabs/spark-nlp/tree/master/examples/python/transformers
